PolyEngine/WindowsGLStandalone/Src/Main.cpp

```diff
@@ -107,11 +107,12 @@ int WINAPI WinMain(HINSTANCE hInstance,
 		{
 			// Run game code here
 			Engine.Update();
-
 			// Temporary visualisation of delta time in window title
-			float dTime = Engine.GetWorld().GetTimeWorldComponent().GetDeltaTime();
-			std::string windowText = "DeltaTime = " + std::to_string(dTime);
-			SetWindowText(hWnd, windowText.c_str());
+			std::stringstream ss;
```

Comment:
Uh, oh. It seems we could use a `toString()` function :P And a complete String implementation+API, right now it's pretty bare-bones.